### PR TITLE
Addresses Issue #7

### PR DESCRIPTION
**Pull request by michael-barlow3**
Added support for multiple definitions as well as `Context` and `Notes` from the Acronyms file.
Updated Readme and added a new test for using custom acronyms.csv filr to test for multiple definitions with context and notes (see tests_wtf_context_notes.py and updated README.md)

### CODE REVIEW

**Comment by michael-barlow3** [comment]
@karlbrown-va - Now tracks context and notes and test cases have been added as well.

File: README.md [hunk 1]
```diff
@@ -43,16 +43,37 @@ $ export DATA_URL=https://raw.githubusercontent.com/department-of-veterans-affai
...
+$ pytest tests_config.py
+$ pytest tests_wtf.py
+```
+
+There's one additional special test for testing out acronyms with multiple definitions as well as `Context` information and `Notes` in the acronyms file. For this a acronyms file has been set up. To run this test the `DATA_URL` needs to be modified to point to the new `acronyms.csv` before running the test
+
+***nix**
+
+```
+$ export DATA_URL=https://raw.githubusercontent.com/department-of-veterans-affairs/wtf-bot/master/test_acronyms.csv
```
Comment (karlbrown-va) [inline README.md:64]: why not just point to the (local) version of test_acronyms.csv? I'm not sure I understand why there is a dependency on a network-hosted version.

Comment (michael-barlow3) [reply README.md:64]: Hadn't thought of using the local version but the original test cases pointed to the online version

**Comment by michael-barlow3** [comment]
Checking now.

Thank you
Mike Barlow
Technical Architect
Clinical Ancillary Product Line
EPMO Health Portfolio PMO/TM

Lancaster, Pa
(732) 682-8226 | M-F 8:30-5:00pm ET
***@***.******@***.***>
***@***.******@***.***>

From: Karl Brown ***@***.***>
Sent: Friday, June 4, 2021 11:58 AM
To: department-of-veterans-affairs/wtf-bot ***@***.***>
Cc: Barlow, Michael W. (GovernmentCIO) ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/wtf-bot] Addresses Issue #7 (#15)


Running the tests fails:

_____________________________________ test_multi_def_with_context_and_note_payload _____________________________________



client = <FlaskClient <Flask 'wtf'>>



    def test_multi_def_with_context_and_note_payload(client):

        data = {'text': 'foo','token': TEST_TOKENS[0]}

        r = client.post(ROUTE, data=data)

        # print (r.data)

>       assert b'foo\n - For Obfuscating Obvious Information\n\t- This is a sample context\n\t- This is a sample note; \n - For Obfuscating Obvious Information 2\n\t- This is a sample context\n\t- This is a sample note; \n - For Obfuscating Obvious Information 3\n\t- This is a sample context\n\t- This is a sample note' in r.data

E       assert b'foo\n - For Obfuscating Obvious Information\n\t- This is a sample context\n\t- This is a sample note; \n - For Obfus... is a sample note; \n - For Obfuscating Obvious Information 3\n\t- This is a sample context\n\t- This is a sample note' in b"\n        Entry for 'foo' not found! Acronyms may be added at\n        https://github.com/department-of-veterans-affairs/acronyms\n<https://github.com/department-of-veterans-affairs/acronyms/n>        "

E        +  where b"\n        Entry for 'foo' not found! Acronyms may be added at\n        https://github.com/department-of-veterans-affairs/acronyms\n<https://github.com/department-of-veterans-affairs/acronyms/n>        " = <WrapperTestResponse 136 bytes [200 OK]>.data



tests_wtf_context_notes.py:35: AssertionError

Can you check this?

-

**Review by karlbrown-va — APPROVED** [review]
I've discussed and reviewed these changes. They add a better interface to WTF bot to reveal a bit more information and have better formatting for multiple result returns. The tests all pass if you follow the instructions (once merged, they will).

I'm working on another branch to get automated tests running, but I think this one can be merged.

**Review by batemapf — APPROVED** [review]

**Pull request merged**